docs/rfc/007-endpoint-errors.md

```diff
@@ -0,0 +1,137 @@
+# [RFC] Conjure Endpoint Errors
+
+**Date:** 2020/03/19
+
+**Stakeholders:**
+* *Anyone that produces or consumes conjure-generated code via Dialogue or Undertow.*
```

Comment:
It impacts everyone, although it may not be a requirement that every implementation adds generator support
```suggestion
* *Anyone that produces or consumes conjure-generated code.*
```